parcels/grid.py

```diff
@@ -104,6 +108,12 @@ def eval(self, x, y):
         v = self.V.eval(x, y)
         return u, v
 
+    def reversetime(self):
+        self.U.dtsign = -1.
+        self.V.dtsign = -1.
+#        self.W.dtsign = -1.  # uncomment line once vertical-vels are implemented
+        return self
+
```

Comment:
For convenience, how about adding a `reverse` or even `reversetime` argument to the signature of `grid.execute()` that calls this if `reverse=True`? Or we could even move the setting of `dtsign` into `exeute()` altogether, since we currently have no way of cleanly deactivating the "reverse" mode?